speed/inrix/sf_inrix_xd_realtime.py

```diff
@@ -1,31 +1,40 @@
 import argparse
-import json
 import tomllib
 from pathlib import Path
+import json
```

Comment:
this one must be a weird typo/edit since it moved the json to out of order. Suggest reverting this change.